### DISCUSSION TITLE
New Complex Components

### DISCUSSION THREAD

**Discussion by djom202** [twbs/bootstrap#40570] [Ideas]
Hi there,

I've been using this tool since version 1, so I know it is great and very versatile for all types of projects. However, I've always wondered why more complex components are not created. While looking for how I could create new ones, I found a site called [SyncFunction](https://ej2.syncfusion.com/home/vue.html?_gl=1*ehp9xs*_gcl_au*MzIxNDM5OTA5LjE3MTg4MTEwMTg.*_ga*MjAxOTc5MTA1MC4xNzE4NjgyMTg3*_ga_41J4HFMX1J*MTcxODgxNTk0Ni40LjEuMTcxODgxNjA1My4wLjAuMA..*_ga_R3Q3BGFFC9*MTcxODgxNjAzMy4xLjAuMTcxODgxNjAzMy42MC4wLjA.&_ga=2.7949173.632610207.1718816033-2019791050.1718682187) that offers a wide range of components. The idea is not to copy them, but I feel that Bootstrap has the potential and maturity to include more complex components. From my perspective, many of these examples are simple, but with each new project, they need to be rebuilt, even though they could be part of your repository for easy insertion. The idea is not to lose the concept of reusable components with these new components that we, as developers, build.

Therefore, I invite you to take a detailed look at these new components, as they could greatly enhance the excellent work you have been doing.

Thank you!

<img width="1159" alt="image" src="https://github.com/twbs/bootstrap/assets/1834808/926c0390-4feb-427a-aacb-23f4b0542a91">

<img width="1062" alt="image" src="https://github.com/twbs/bootstrap/assets/1834808/32580267-fb15-4d26-b2d8-4f8c2a613de7">

**Comment by ajiho** [comment]
Is it open source and free?
